spock-specs/src/test/groovy/org/spockframework/smoke/mock/JavaSpies.groovy

```diff
@@ -113,6 +113,18 @@ class JavaSpies extends Specification {
     result == "singing, singing"
   }
 
+  def "sping on concrete instances can use partial mocking"() {
```

Comment:
You could link to https://github.com/spockframework/spock/issues/771 here (`@Issue`)